frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -172,7 +174,11 @@ class Dashboard extends Component {
         this.setEditing(this.props.dashboard);
       }
       if (addCardOnLoad != null) {
-        addCardToDashboard({ dashId: dashboardId, cardId: addCardOnLoad });
+        addCardToDashboard({
+          dashId: dashboardId,
+          cardId: addCardOnLoad,
+          tabId: this.props.dashboard.ordered_tabs[0]?.id ?? null,
```

Comment:
When adding a card on load (e.g. from a new/saved question) default to putting that card on the first tab.